auth/oidc_test.go

```diff
@@ -207,71 +207,38 @@ func TestOIDCUsername(t *testing.T) {
 
 }
 
-func TestOIDCProvider_InitOIDCClient(t *testing.T) {
-	defer base.SetUpTestLogging(base.LevelInfo, base.KeyAuth)()
-
-	clientID := "SGW-TEST"
-	callbackURL := "http://sgw-test:4984/_callback"
-
-	tests := []struct {
-		Name             string
-		Provider         *OIDCProvider
-		ErrContains      string
-		ExpectOIDCClient bool
-	}{
-		{
-			Name:        "nil provider",
-			ErrContains: ErrMsgNilProvider,
-		},
-		{
-			Name:        "empty provider",
-			Provider:    &OIDCProvider{},
-			ErrContains: "Issuer not defined",
-		},
-		{
-			Name: "unavailable",
-			Provider: &OIDCProvider{
-				Issuer: "http://127.0.0.1:12345/auth",
-			},
-			ErrContains: ErrMsgUnableToDiscoverConfig,
-		},
-		{
-			Name: "valid provider",
-			Provider: &OIDCProvider{
-				ClientID:    clientID,
-				Issuer:      "https://accounts.google.com",
-				CallbackURL: &callbackURL,
-			},
-			ExpectOIDCClient: true,
-		},
-	}
-
+func TestInitOIDCClient(t *testing.T) {
```

Comment:
Is there any test coverage for concurrent Verify() and SetVerifier() calls?  Mutex operations are a common source of deadlocks/races, and we'd want to ensure that's covered in the tests.

Reply:
Added a concurrent client initialization test for SetVerifier() calls.